Packages/cdms2/Lib/dataset.py

```diff
@@ -283,9 +283,10 @@ def openDataset(uri,mode='r',template=None,dods=1,dpath=None, hostObj=None):
     uri = string.strip(uri)
     (scheme,netloc,path,parameters,query,fragment)=urlparse.urlparse(uri)
     if scheme in ('','file'):
+        if uri[:7].lower() == "file://":
```

Comment:
> but if it starts with file:// then path and root are both empty strings.

@doutriaux1 path should not be empty if you parse it via urlparse. I just did a local test to confirm this.